cmd/util/ledger/migrations/cadence.go

```diff
@@ -345,7 +347,71 @@ func (m *IssueStorageCapConMigration) Close() error {
 	return nil
 }
 
-var _ AccountBasedMigration = &IssueStorageCapConMigration{}
+func (m *IssueStorageCapConMigration) issueAccountCapabilities(
```

Comment:
This method was copied over and modified from Cadence side. Need to access the storage, etc. so having it here makes more sense.
The old method can be remove from cadence.

Reply:
We might need to re-incorporate the logic from https://github.com/onflow/cadence/pull/3516 once it's merged, or move the changes here to Cadence.